cinema/urls.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path(
+        "movies/",
+        movie_list,
+        name="movie-list"
+    ),
+    path(
+        "movies/<int:pk>/",
+        movie_detail,
+        name="movie-detail"
+    )
+]
+
+app_name = 'cinema'
```

Comment:
add blank line here

Reply:
There is a blank line, it's just not shown here